examples/lib/defaultHtmlStepUi.js

```diff
@@ -26,35 +26,40 @@ function DefaultHtmlStepUi(_sequencer, options) {
     step.ui =
       '\
       <div class="container">\
+      <div class="panel panel-default">\
+      <div class="panel-heading">\
+        <h3 class="panel-title">' +  
+        '<span class="toggle">' +step.name + '<span class="caret"></span></h3>\
+        </div>\
+      </h3>\
+    <div class="panel-body cal collapse in">\
     <div class="row step">\
     <form class="input-form">\
     <div class="col-md-4 details">\
     <h3>\
-    <span class = "toggle">' +step.name + ' <i class="fa fa-caret-up toggleIcon" aria-hidden="true"></i></span>' +
-    '<span class="load-spin" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></span>' +
+    <span class="load-spin" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></span>' +
     '</h3><div class="cal collapse in"><p><i>"'+
       (step.description || "") +
       '</i></p></div>\
     </div>\
-    </form>\
     <div class="col-md-8 cal collapse in step-column">\
       <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
       <div class="step-image">\
-        <a class="cal collapse in"><img alt="" class="img-thumbnail step-thumbnail"/></a>\
+        <a class="cal collapse in"><img alt="" id="myimg" class="img-thumbnail step-thumbnail"/></a>\
       </div>\
     </div>\
     </div>\
     </div>\
```

Comment:
Ok see. This is the ending of a div element. The form ends after this ends. The form should also start before this ends. The form will have to contain the whole div inside it. Please do so if you haven't.